dataactcore/models/views.py

```diff
@@ -0,0 +1,16 @@
+from sqlalchemy import MetaData, Table
+from dataactcore.interfaces.db import GlobalDB
+
+
+class SubmissionUpdatedView:
```

Comment:
General question: does it matter if we instantiate the view object versus making the `__init__` into another, static function to return the view? Wondering out of curiosity since some people in the python community say don't use static functions, some say do.